print-nfs.c

```diff
@@ -1668,6 +1668,10 @@ interp_reply(netdissect_options *ndo,
 					ND_PRINT(" <%s>",
 						tok2str(nfsv3_writemodes,
 							NULL, EXTRACT_BE_U_4(dp + 1)));
+
+					/* write-verf-cookie */
+					ND_TCHECK_8(dp + 2);
+					ND_PRINT(" verf %" PRIx64, EXTRACT_BE_U_8(dp + 2));
 				}
```

Comment:
The above reads as "4 bytes addressed as dp[1], dp[2], dp[3] and dp[4] stand for the write mode" and "8 bytes addressed as dp[2], dp[3], dp[4], dp[5], dp[6], dp[7], dp[8] and dp[9] stand for the cookie". Clearly there is a 3-byte overlap, could you double-check and confirm?